packages/@lwc/ssr-compiler/src/compile-template/transformers/element.ts

```diff
@@ -263,6 +275,7 @@ export const Element: Transformer<IrElement | IrExternalComponent | IrSlot> = fu
 
     return [
         bYield(b.literal(`<${node.name}`)),
+        ...[bConditionallyYieldDanglingSlotName()],
```

Comment:
```suggestion
        bConditionallyYieldDanglingSlotName(),
```
Creating and then spreading an array with one element is unnecessary.

Reply:
Good catch, thx